### PR TITLE
fix: update header (CXSPA-1067)

### PR DESCRIPTION
**Pull request by kimhw0630**
Closes: https://jira.tools.sap/browse/CXSPA-1067

changed header tag to div and added aria-labelledby & role

### CODE REVIEW

File: projects/storefrontlib/cms-components/product/product-list/product-facet-navigation/active-facets/active-facets.component.html [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <ng-container *ngIf="facetList$ | async as facetList">
   <!-- TODO:(CXSPA-1695) for next major release update below feature flags -->
   <ng-container *cxFeatureLevel="'5.2'">
-    <h2 *ngIf="facetList.activeFacets && facetList.activeFacets.length > 0">
+    <div id="groupName" *ngIf="facetList.activeFacets && facetList.activeFacets.length > 0">
```
Comment (justinlee01) [inline projects/storefrontlib/cms-components/product/product-list/product-facet-navigation/active-facets/active-facets.component.html]: `groupName` does not seem to be a sufficient unique ID string...

Comment (kimhw0630) [reply projects/storefrontlib/cms-components/product/product-list/product-facet-navigation/active-facets/active-facets.component.html]: yap... i will change to `cx-active-facets-groupName`

**Review by justinlee01 — DISMISSED** [review]
If there are multiple of this component, `cx-active-facets-groupName` isn't unique enough either. But I will approve until I see evidence there is.

File: projects/storefrontlib/cms-components/product/product-list/product-facet-navigation/active-facets/active-facets.component.html (hunk 1 above)
Comment (developpeurweb) [reply projects/storefrontlib/cms-components/product/product-list/product-facet-navigation/active-facets/active-facets.component.html]: "groupName" is what came with the example suggestion, but please follow the standard naming conventions for IDs. Ask a senior angular dev if in doubt.

**Review by developpeurweb — DISMISSED** [review]
Works well with VO and JAWS

**Review by Zeyber — APPROVED** [review]

**Pull request merged**